accessman_test.go

```diff
@@ -1,7 +1,6 @@
 package lnd
 
```

Comment:
Seems like there are still some places where we could replace the context call ?

For example: `TestEtcdElector` or `TestChainWatcherRemoteUnilateralClosePendingCommit` ? How did you replace all those calls in this commit ?

Reply:
I fixed this and several other instances inside `lntest/` dir.